pallets/manta-sbt/src/lib.rs

```diff
@@ -299,17 +316,30 @@ pub mod pallet {
         /// Mints a zkSBT
         ///
         /// `TransferPost` is posted to private ledger and SBT metadata is stored onchain.
+        /// `signature` parameter can be used to relay a tx.
         #[pallet::call_index(0)]
         #[pallet::weight(<T as pallet::Config>::WeightInfo::to_private())]
         #[transactional]
         pub fn to_private(
-            who: OriginFor<T>,
+            origin: OriginFor<T>,
+            signature: Option<SignatureInfoOf<T>>,
```

Comment:
I think we need `mintId` as we also have different project of pomp.

Reply:
If we are storing data in the metadata we could also include the mint-id into the metadata